packs/feats/patron-reborn.json

```diff
@@ -31,7 +31,11 @@
         "rules": [],
         "subfeatures": {
             "senses": {
-                "darkvision": {}
+                "darkvision": {},
+                "truesight": {
+                    "acuity": "imprecise",
```

Comment:
This imprecise acuity is here despite the UI showing precise, and acuity not really even being an editable field for it.

Reply:
That does seem like a bug, given that senses.ts:42 suggests it would be forced to precise.